app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -278,6 +279,22 @@ def volumes
     end
   end
 
+  # Store VPC volume profiles in cloud_volume_types table. Use name for the ems_ref and name. Use the family as description.
+  # @return [void]
+  def cloud_volume_types
+    $ibm_cloud_log.info('Starting cloud volume types persistance.')
```

Comment:
Technically this isn't where persistence is happening, this is the parsing step.

Also this logging might be confusing since no other collection logs this it will look like _only_ cloud volume types are being parsed.